woocommerce/woocommerce
# Add payment icons right next to the Discover other payments link

### All Submissions:






### Changes proposed in this Pull Request:



Partially closes #33626 .

This PR adds payment icons right next to the Discover other payments link.

### How to test the changes in this Pull Request:

Case 1: WC Pay is not installed and setup

1. Start with a fresh install.
2. Complete OBW with a WC Pay eligible country.
3. Navigate to `WooCommere -> Settings -> Payments`
4. You should see Stripe and Paypal icons when country is the US

Case 2: WC Pay is setup

1. Start with a fresh install
2. Complete OBW with a WC Pay eligible country.
3. Set up WCPay OR open `./includes/class-wc-payment-gateway-wcpay.php` from WooCommerce Payments plugin. Return `false` from the `needs_setup()` method. This will make WooCommerce think that WC Pay is already set up.
3. Navigate to `WooCommere -> Settings -> Payments`
4. You should see Klarna, Paypal, Afterpay, Amazon Pay, and Affirm icons when country is the US


### Screenshots

![Screen Shot 2022-07-05 at 8 29 37 PM](https://user-images.githubusercontent.com/4723145/177462121-d95caeea-bce5-4e8c-aa38-d17a02275b3b.jpg)

### Other information:




### FOR PR REVIEWER ONLY:
